lib/authentication/authentication.js

```diff
@@ -77,7 +77,7 @@ exports.getAuthenticator = function getAuthenticator(connectionConfig)
   }
   else if (auth == authenticationTypes.EXTERNAL_BROWSER_AUTHENTICATOR)
   {
-    return new auth_web(connectionConfig.host);
+    return new auth_web(connectionConfig.host, undefined, undefined, connectionConfig.getBrowserActionTimeout());
```

Comment:
in production code second and third parameter are always undefined - let's move them to the end since timeout is obligatory here